packit_service/constants.py

```diff
@@ -74,6 +75,53 @@
 
 REQUESTED_PULL_REQUEST_COMMENT = "/packit"
 
+SRPM_BUILD_DEPS = [
```

Comment:
How did we get this list? So, it's packit + usual build requirements? Is there a way to make this configurable?

* It would be nice to get rid of the scaling problems of Sandcastle dependences by moving the SRPM build to Copr.
* Installing all these takes some time.

Reply:
> How did we get this list? So, it's packit + usual build requirements?

Yes, it's [these](https://github.com/packit/sandcastle/blob/3bd64e0812e3981b5462601e049471854eec433a/files/install-rpm-packages.yaml#L6)

> Is there a way to make this configurable?
>
> -  It would be nice to get rid of the scaling problems of Sandcastle dependences by moving the SRPM build to Copr.
>   - Installing all these takes some time.

I think we discussed this in the architecture meeting and the outcome was that the initial implementation will have the builddeps hardcoded and yes, then we want to achieve making it possible to have them configurable. 
I tested building SRPMs in Copr with some projects (https://copr.fedorainfracloud.org/coprs/lbarczio/test-custom-source/builds/)  with this list of deps and it takes ~3, 4 minutes to build an SRPM including the installation of packages.

Reply:
OK, so let's start with this and improve later on. I agree.